coalib/coala.py

```diff
@@ -27,6 +30,10 @@
 
 
 def main():
+    # Configure logging
+    with open(log_settings_file) as fd:
+        logging.config.dictConfig(json.load(fd))
```

Comment:
I feel like this thing belongs into a function, it's going to be needed in other main functions as well right?